server/server.js

```diff
@@ -1,74 +1,202 @@
 const app = express();
 app.use(express.static(REACT_BUILD_DIR));
 
+//trying the set up from S.O
+const yelp = require('yelp-fusion');
+const { response } = require('express');
+const apiKey = process.env.API_KEY;
+
+// const searchRequest = {
+//     term: 'restaurants',
+//     location: 'Dallas'
+// };
+
+// const searchRequest = {
+//     term: 'restaurants',
+//     location: 'Gunsan'
+// };
+
+const client = yelp.client(apiKey);
+//S.O stuff stops here
+
+/* 'use strict';
+
+const yelp = require('yelp-fusion');
+const client = yelp.client('YOUR_API_KEY');
+
+client.business('gary-danko-san-francisco').then(response => {
+  console.log(response.jsonBody.name);
+}).catch(e => {
+  console.log(e);
+});
+
+'use strict';
+
+const yelp = require('yelp-fusion');
+const client = yelp.client('YOUR_API_KEY');
+
+client.search({
+  term: 'Four Barrel Coffee',
+  location: 'san francisco, ca',
+}).then(response => {
+  console.log(response.jsonBody.businesses[0].name);
+}).catch(e => {
+  console.log(e);
+});
+*/
+
 const PORT = process.env.PORT || 8080;
 app.use(cors());
 app.use(express.json());
 
-//creates an endpoint for the route /api
+
+//creates an endpoint for the route /api;
 app.get('/', (req, res) => {
     res.sendFile(path.join(REACT_BUILD_DIR, 'index.html'));
 });
 
-//create the get request
-app.get('/api/students', cors(), async (req, res) => {
+
+app.get("/api/business/:alias", async (req, res) => {
+    yelpResponse = await client.business(req.params.alias);
+    res.send(yelpResponse.jsonBody)
+    try {
+        const data = await response.json
+        console.log(data);
+    } catch (err) {
+        console.error(err);
+        res.send(err)
+    }
+});
+
+//creates an endpoint for the route /api; this one is beautiful
+app.get("/api/location-search", cors(), async (req, res) => {
+    location = req.query.location;
+    console.log("Line 48 check", location);
+
+    yelpResponse = await client.search({
+        term: 'brunch',
+        location: 'Richardson, TX'
+    });
+    res.send(yelpResponse.jsonBody)
+});
+
+//the get request for the restaurant location for the post with original request method
+// app.get("/api/location-search", async (req, res) => {
+//     const term = req.body.term;
+//     const location = req.query.location;
+//     const url = `https://api.yelp.com/v3/businesses/search?term=brunch&location=richardson, TX`
+
+    
+//     client.search({
+//         term,
+//         location
+//     })
+//    try {
+//     const response = await fetch(url)
+//     const data = await response.json()
+//     console.log(data);
+//     //res.send(data);
+//    } catch (error) {
+//     console.error(error);
+//     res.send(error)
+//    };
+// });
+
+
+
+//yelp api call from S.O, called immedieately instead of on cue
+// client.search(searchRequest)
+//   .then((response) => {
+//     console.log(response.jsonBody);
+//   })
+//   .catch((error) => {
+//     console.log(error);
+//   });
```

Comment:
Leaving these comments out of the PR can help reviewers focus on relevant comments. It's also best to not leave any lingering code until it's ready to be active. You can always recover your commented code through `git` methods